content/docs/hooks-overview.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: एक नज़र में Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+React 16.8 में *Hooks* एक नया फीचर ऐड हुआ है। वे आपको class लिखे बिना state और अन्य React सुविधाओं का उपयोग करने देता हैं।
```

Comment:
*Hooks* naamak एक नया ...

Reply:
वे आपको -> ye aapko

Reply:
class लिखे बिना -> class ka istemaal kiye bina

Reply:
सुविधाओं  -> विशेषताएं

Reply:
हैं। -> hai

Reply:
class ka istemaal kiye bina => kiye is missing

Reply:
हैं -> है

Reply:
@saranshkataria 
I am a bit confused between हैं and है, Can you please give some examples where हैं should be used, and where है should be used.

I personally feel है should be used everywhere.

Reply:
है is mostly suitable everywhere, हैं would be for plural stuff